vendors/twtg/drivers/neon-ds-ts-xx-xx/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "neon-ds-ts-xx-xx",
+  "version": "1.0.0-SNAPSHOT",
+  "description": "NEON Temperature Transmitter driver",
```

Comment:
Temperature Sensor not Temperature Transmitter

Reply:
Corrected